content/espanol/sql_workshop/activity10/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Actividad 10: Aprende el comando SUM"
+description: "Aprende el comando SUM"
+date: 2020-07-07
+prereq: "Comando SELECT, comando WHERE"
+difficulty: "Principiante"
+draft: false
+---
+<!-- Links for javascript and CSS needed for drop down logic -->
+<link rel="stylesheet" href="../default/_default.css" type="text/css"></link>
+<link rel="stylesheet" href="../default/_type.css" type="text/css"></link>
+<script type="text/javascript" src="../default/_default.js"></script>
+<script type="text/javascript" src="../default/_type.js"></script>
+<script type="text/javascript" src="../default/alasql.js"></script>
+<script type="text/javascript" src="../default/db.js"></script>
+<link rel="stylesheet" href="_activity10.css" type="text/css"></link>
+<script type="text/javascript" src="_activity10.js"></script>
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/UCxPAE0xw3I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+## Regresa los Tótems Legendarios del planeta Fun al Lider
+
+Haz encontrado exitosamente los dos Tótems Legendarios del planeta Fun. El Líder te agradece por encontrar su tesoro perdido.
+
+### El Líder revela un secreto:
+{{% notice info %}}
+Los Tótems Legendarios del planeta Fun no pueden liberar todo su potencial hasta que sean unidos juntos. Unicamente pueden ser unidos por el "Inventor".
+
+Tu siguiente paso es encontrar a la persona que pueda combinar ambos Tótems Legendarios del planeta Fun. El "Inventor" solo va a aparecer si organizas una gran fiesta.
+
+{{% /notice%}}
+
+## Tarea 10: Ayuda a los habitantes a organizar la fiesta.
+The whole planet is invited! Help the citizens prepare for the party by finding out how many guests will be attending!
+
+Aprendamos el comando `SUM`
+
+![Explain](assets/sum.png)
+
+#### Usando la base de datos  'planet'(planeta) y el comando `SUM()`, encuentra el numero total de invitados que van a asistir a esta increíble fiesta.
+
+{{% notice tip %}}
+Usa el comando de la actividad 1 para mostrar toda la base de datos.
+
+* Pista 1: El comando es muy similar al ejemplo de arriba y el ejemplo del ejercicio de `AVG()` (tarea anterior).
+* Pista 2: La columna 'population'(población) contiene información sobre el número de individuos por ciudad.
+{{% /notice%}}
+
+<!-- SQL Type In Activity -->
+
+{{< rawhtml >}}
+<div class="content_scaler">
+  <div class="terminal_div" id="terminal_div">
+    <div class = "outer">
+      <h3 id = "commands" contenteditable="true" onclick="placeholder()">Escribe el comando aquí</h3>
+    </div>
+    <div class = "prev">
+      <h3 id = "prev"></h3>
+    </div>
+    <div style="clear: both;"></div>
+    <button class="button button1" onclick="sql()"> Enter </button>
```

Comment:
Translate Enter (and similarly elsewhere)

Reply:
I think "Enter" is valid in spanish too.